packages/legend-studio/src/stores/DSLMapping_LegendStudioPlugin_Extension.ts

```diff
@@ -162,4 +200,28 @@ export interface DSLMapping_LegendStudioPlugin_Extension
    * Get the list of actions for mapping test query editor.
    */
   getExtraMappingTestQueryEditorActionConfigurations?(): MappingTestQueryEditorActionConfiguration[];
+
+  /**
+   * Get the list of extra mapping elements by using type and Id.
+   */
+  getExtraMappingElementByTypeAndIds?(): MappingElementGetter[];
+
+  /**
+   * Get the list of extra class mapping source drivers.
+   */
+  getExtraClassMappingSourceDrivers?(): ClassMappingSourceDriver[];
+
+  /**
+   * Get the list of extra source types and names.
+   */
+  getExtraSourceTypesAndNames?(): SourceTypeAndNameGetter[];
+
+  /**
+   * Get the list of extra property mapping editors.
+   */
+  getExtraPropertyMappingsEditors?(): PropertyMappingsEditor[];
+
+  getExtraCompilationErrorRevealers?(): CompilationErrorRevealer[];
+
+  getExtraBackDropElements?(): BackDropElementGetter[];
```

Comment:
--> `getExtraInstanceSetImplementationBlockingErrorCheckers`. The documentation for this is something like `Get the list of checkers for errors in the specified instance set implementation that could potentially block the user interaction until fixed (e.g. parser error in some property mapping transform lambda editor)`